src/sass/style.scss

```diff
@@ -329,8 +329,8 @@ table {
 
 html {
   background-image: url('img/denim.png');
-  color: $lightest;
   background-color:  $dark;
```

Comment:
of course you can remove this line now :) Thanks for being that quick updating the PR!